internal/host/common.go

```diff
@@ -39,6 +39,8 @@ const (
 	statusInfoAbortingDueClusterErrors       = "Host is part of a cluster that failed to install"
 	statusInfoInstallationTimedOut           = "Host failed to install due to timeout while starting installation"
 	statusInfoInstallationInProgressTimedOut = "Host failed to install because its installation stage $STAGE took longer than expected $MAX_TIME"
+
```

Comment:
please remove empty line

Reply:
made the empty line to distinguish between two types of messages.